pylabrobot/liquid_handling/backends/hamilton/STAR.py

```diff
@@ -4046,10 +4050,17 @@ async def get_core(self, p1: int, p2: int):
     if not 1 <= p2 <= self.num_channels:
       raise ValueError(f"channel_2 must be between 1 and {self.num_channels}")
 
+    deck_size = self.deck.get_size_x()
+    if deck_size == STARLET_SIZE_X:
+      xs = "07975"
+    elif deck_size == STAR_SIZE_X:
+      xs = "13375"
```

Comment:
this we should be able to get from `deck.get_size_x()` right?

Reply:
one would think! But these actually turn out to be different values, perhaps because of excess padding by hamilton.